packages/@react-types/calendar/src/index.d.ts

```diff
@@ -62,7 +62,12 @@ export interface CalendarPropsBase {
    * Controls the behavior of paging. Pagination either works by advancing the visible page by visibleDuration (default) or one unit of visibleDuration.
    * @default visible
    */
-  pageBehavior?: PageBehavior
+  pageBehavior?: PageBehavior,
+  /**
+   * Whether the calendar should always display 6 weeks.
+   * @default false
+   */
+  isFixedWeeks?: boolean
```

Comment:
The team is currently discussing whether this should be supported for the React Spectrum side of things as well but perhaps this should be moved to the RAC Calendar props specifically for now